pkg/server/apis/v1alpha1/types.go

```diff
@@ -2,23 +2,24 @@ package v1alpha1
 
 import (
 	core_v1 "k8s.io/api/core/v1"
+	cmd_api "k8s.io/client-go/tools/clientcmd/api"
 )
 
 // Tenant contains information about tenant
 type Tenant struct {
 	// Metadata contains metadata information about tenant
-	Metadata TenantMetadata `json:"metadata"`
+	Metadata Metadata `json:"metadata"`
 	// Spec contains tenant spec
 	Spec TenantSpec `json:"spec"`
 }
 
-// TenantMetadata contains metadata information about tenant
-type TenantMetadata struct {
-	// Name is the name of tenant, unique.
+// Metadata contains metadata information
+type Metadata struct {
```

Comment:
What about use k8s metadata?

Reply:
I think there is a litter different between them:
- no `Description` field in k8s metadata.
- some field in k8s metadata we don't need, e.g. `UID` `ResourceVersion` `Generation` `Initializers` etc.